__init__.py

```diff
@@ -23,36 +22,6 @@ def pre_init_plugin():
     pkg_resources.working_set.add_entry(extra_libs_path)
 
 
-def import_ee():
-    """This is a wrapper of the Google Earth engine library for the
-    purpose of initializing or starting ee authentication when the
-    user or the plugin import ee library.
-    """
-    # we can now import the libraries
-    # Work around bug https://github.com/google/earthengine-api/issues/181
-    import httplib2
-
-    from ee_plugin.ee_auth import authenticate
```

Comment:
Removing this file ee_auth.py was a bad idea, we still need EE authentication code. It does not have to be called as implicitly as it was done here, but it still must be called somewhere on the first start of the plugin if EE Python API is not authenticated yet. I'm reverting the file deletion to call when the plugin is started and there is no authentication info available for EE.

Reply:
Actually, ee.Authenticate(auth_method='localhost') seems to do the same as this code, so it's fine to remove it.